src/Common/src/System/Security/Cryptography/DerSequenceReader.cs

```diff
@@ -194,6 +194,30 @@ internal string ReadOidAsString()
             return builder.ToString();
         }
 
+        internal byte[] ReadBitString()
+        {
+            // Bitstrings are primitives with tag and length, followed by a number
+            // of unused bits in the last byte, followed by the 
+            EatTag(DerTag.BitString);
+            int length = EatLength();
+
+            if (length == 0)
+                return Array.Empty<byte>();
```

Comment:
Is length 0 valid?  "If a bitstring value has no 1 bits, then an encoder shall encode the value with a length of 1 and an initial octet set to 0." (X.690-2015 11.2.2)